### PR TITLE
To-Do-List Interactive

### PR DESCRIPTION
**Pull request by diegoh40**
In this project we implemented the modules, we also developed the method to update the status of 'completed' true or false in conjunction with the checkbox of each task. Then we implemented the 'Clear all completed' and correctly configured the local storage updating it for a good user experience.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,92 +1,24 @@
 import './style.css';
+import StorageL from './modules/storage.js';
```
Comment (Kossi-stack) [inline src/index.js:2]: - [ ] Kindly write a function that will reorder a selected item (as drag-and-drop). This is part of the main requirement.

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @diegoh40,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Required Changes ♻️
 
 _Check the comments under the review for more changes_.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/index.js (hunk 1 above)
Comment (diegoh40) [reply src/index.js:2]: Hi i think that is optional drag and drop it not neccesary , please check

**Comment by diegoh40** [comment]
I think the code reviewer did not notice the optional requirements, the remark he makes, I did not make it because it was optional to pass this test. so again I submit my pull request to please take this into consideration.
image

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @diegoh40 , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**